lib/src/components/month_view_components.dart

```diff
@@ -149,8 +146,6 @@ class FilledCell<T> extends StatelessWidget {
                   physics: BouncingScrollPhysics(),
                   child: Column(
                     mainAxisSize: MainAxisSize.min,
-                    crossAxisAlignment: CrossAxisAlignment.center,
-                    mainAxisAlignment: MainAxisAlignment.start,
```

Comment:
Why we removed `MainAxisAlignment` and `CrossAxisAlignment`?

Reply:
Values we provided as arguments are default values for Column. So whether we provide it or not doesn't matter.